android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/adding/AddingFragment.java

```diff
@@ -206,6 +206,17 @@ private Cashflow createCashFlow() {
         return null;
     }
 
+    private void resetErrorMessages() {
+        List<TextInputLayout> textfields = new ArrayList<>();
+        textfields.add(storeLayout);
+        textfields.add(amountLayout);
+        textfields.add(dateTextLayout);
+        textfields.add(categoriesDropdownLayout);
+        textfields.add(addressLayout);
```

Comment:
```suggestion
        List<TextInputLayout> textfields = List.of(storeLayout, amountLayout, dateTextLayout, categoriesDropdownLayout, categoriesDropdownLayout, addressLayout);
```
 small refactoring, untested but this should be the syntax